src/joystick/joystick_manager.h

```diff
@@ -0,0 +1,96 @@
+#include "roboteam_msgs/RobotCommand.h"
+#include "roboteam_utils/Vector2.h"
+#include "roboteam_msgs/RoleDirective.h"
```

Comment:
do we need role directive? geometryFieldSize? world_analysis? geometry data?

Reply:
I am not sure I will look at it. For now I just left the headers that were already there